ui/react-ui/src/components/app/App.js

```diff
@@ -0,0 +1,22 @@
+import './App.css';
+import {BrowserRouter as Router, Route} from 'react-router-dom'; 
+import UserTasks from '../routes/userTasks/UserTasks.js';
+import Home from '../routes/home/Home.js';
+ 
+function App() {
+  return (
+    <div className="App">
+      <Router>
+        <Route path='/' exact component={Home} />
+        {/* Jak */}
```

Comment:
Routes implemented, and thanks for creating commented out placeholders for other  devs.